src/plugins/tests.jl

```diff
@@ -19,44 +27,83 @@ Sets up testing for packages.
 @plugin struct Tests <: FilePlugin
     file::String = default_file("test", "runtests.jl")
     project::Bool = false
+    aqua::Bool = false
```

Comment:
Thanks for adding this! 

In many of my projects, I set aqua but with a different set of default kwargs (often `ambiguities=false`). Perhaps we could add an additional field to the `Tests` struct that allows a user to set those kwargs up front? 
e.g., 
```suggestion
    aqua::Bool = false
    aqua_kwarg_str::String = ""
```
and then (below)
```
kwargs = isempty(p.aqua_kwarg_str) ? "" : "; $(p.aqua_kwarg_str)"
Aqua.test_all($pkg$kwargs)
```

(or a sleeker version that passes the params through as a NamedTuple! But this easy version would be sufficient for my needs)

Reply:
done in the next commit :)

Reply:
thanks!